tests/functional.js

```diff
@@ -14,6 +14,7 @@ define([
   './functional/sync_v2_sign_in',
   './functional/sync_v2_force_auth',
   './functional/sync_v3_sign_up',
+  './functional/amo_sign_up',
```

Comment:
Can you remove this line from `functional.js` and add it to `functional_oauth.js` instead?